src/main/java/frc/robot/RobotContainer.java

```diff
@@ -88,7 +88,11 @@ public class RobotContainer implements Logged {
   int[] rotationColor;
   int[] XTranslationColor;
   int[] YTranslationColor;
-
+  int[] previousRotationColor;
+  int[] previousXTranslationColor;
+  int[] previousYTranslationColor;
+  int[] previousRotationCorrect;
+  int[] desiredRotationCorrect;
```

Comment:
You don't need this, our "desiredRotationCorrect"/current rotation is just the existing variable: rotationCorrect